README.md

```diff
@@ -11,12 +11,40 @@ ffwd decorates the received metrics with system-wide tags or attributes. By doin
 
 __Head over to https://spotify.github.io/ffwd/ for documentation.__
 
-# Building
+
+# Quick Start
+
+ffwd can be started quickly with docker. This can be useful to run locally when troubleshooting metrics with your service.
+
+```bash
+docker run -it -p 19000:19000 -p 8080:8080 spotify/ffwd:latest
```

Comment:
Should we add the udp ports to the run command?

Reply:
good catch - i'll add `19091` since that is what apollo is using